flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/slotmanager/DefaultResourceAllocationStrategy.java

```diff
@@ -347,10 +367,26 @@ && canFulfillRequirement(effectiveProfile, remainResource)) {
         }
     }
 
-    private void tryFulFillRedundantResources(
-            ResourceProfile requiredRedundantResource,
+    private boolean isRequiredResourcesFulfilled(
+            ResourceProfile availableResourceProfile, ResourceProfile totalResourceProfile) {
```

Comment:
We already have a class variable `totalResourceProfile`, It's better to not use this in method arguments.

```suggestion
            ResourceProfile resourceAvailable, ResourceProfile resourceInTotal) {
```